client/js/components/support/DpSupportCard.vue

```diff
@@ -26,8 +26,7 @@ All rights reserved
       v-if="email"
       v-text="email" />
     <div
-      v-if="reachability.service"
-      class="u-pb-0_75">
+      v-if="reachability.service">
```

Comment:
v-if should be ordered first

Reply:
the class attribute is not needed any more, it was removed

Reply:
Oh, right, my mistake